src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -1238,6 +1228,22 @@ protected void ExecuteIndexing()
             }
         }
 
+        private void NotifyAboutCompletedBatch(bool didWork)
+        {
+            // this side-by-side index will be replaced in a second, notify about indexing success
+            // so we know that indexing batch is no longer in progress
```

Comment:
This comment should not be here but when it's called

Reply:
done - sorry